conf/config.go

```diff
@@ -122,40 +122,49 @@ func LoadConfig(mainConfigFile string, fallbackConfigFile string) (*MenderConfig
 		return nil, loadErr
 	}
 
+	log.Debugf("Loaded %d configuration file(s)", filesLoadedCount)
+
 	if filesLoadedCount == 0 {
 		log.Info("No configuration files present. Using defaults")
 		return config, nil
 	}
 
-	if config.Servers == nil {
-		if config.ServerURL == "" {
+	log.Debugf("Loaded configuration = %#v", config)
+
+	return config, nil
+}
+
+// VerifyServerConfig verifies the Servers fields in the configuration
+func (c *MenderConfig) VerifyServerConfig() error {
```

Comment:
Nitpick: I wouldn't make the public interface so specific; what about simply `Validate() error`? This also leaves room for checking other possible illegal configuration in the future.

Reply:
This is a good idea, thanks.